eo-maven-plugin/src/test/java/org/eolang/maven/optimization/OptCachedTest.java

```diff
@@ -158,4 +196,23 @@ private static XML program(final ZonedDateTime time) {
             ).xmlQuietly()
         );
     }
+
+    /**
+     * Generates EO program for tests with specified time and content.
+     * @param time Time.
+     * @param something String.
+     * @return XML representation of program.
+     */
+    private static XML programWithSomethings(final ZonedDateTime time, final String something) {
```

Comment:
@Yanich96 Maybe we can overload the already existed `private static XML program(final ZonedDateTime time)` method? In this case we won't need to repeat the same code twice.

Reply:
@volodya-lombrozo I think existed method doesn't show needed results. I did the same program with a different context, so I can show that time verification is not reliable

Reply:
@Yanich96 I understood your intentions here. I just suggest to apply [method overloading](https://en.wikipedia.org/wiki/Function_overloading):
```java
    private static XML program(final ZonedDateTime time) {
        return OptCachedTest.program(time, "same");
    }

    private static XML program(final ZonedDateTime time, final String content) {
        return new XMLDocument(
            new Xembler(
                new Directives()
                    .add("program")
                    .attr("name", "main")
                    .attr("time", time.format(DateTimeFormatter.ISO_INSTANT))
                    .attr("something", content)
                    .up()
            ).xmlQuietly()
        );
    }
```